types/rpc-handler.ts

```diff
@@ -60,26 +61,113 @@ export class RPCHandler implements HandlerInterface {
     this.getNetworkName.bind(this);
     this.getNetworkRpcs.bind(this);
     this.testRpcPerformance.bind(this);
+    this.consensusCall.bind(this);
+    this.getFirstAvailableRpcProvider.bind(this);
+    this._rpcService = new RPCService(this);
   }
 
   /**
-   * Loops through all RPCs for a given network id and returns a provider with the first successful network.
+   * @NOTICE # This method is intended for read operations only.
+   * Performing write operations with this method is dangerous as it is executed and confirmed by potentially all supported network providers.
```

Comment:
@Keyrxng I don't understand what's exactly dangerous. Could you provide an attack vector?

Reply:
I mean dangerous in the sense that we will ask potentially 10+ providers to execute this transaction for us and return the response. 

If a user is not aware and puts a token transfer through for example then it would execute n times wouldn't it? 

What they probably meant to do was transfer through the `provider` and verify the `txHash` with `consensusCall`.

Reply:
> If a user is not aware and puts a token transfer through for example then it would execute n times wouldn't it?

No, the EOA's nonce will be the same, some of the nodes will put a new tx in the mempool, some of the pools won't sync the mempool if tx with the same nonce already exists in other nodes' mempools, some of the laggy nodes will discard (i.e. remove from the mempool) tx after some time

Reply:
Ahh yeah I understand my confusion now, I appreciate the feedback.